### DISCUSSION TITLE
Add more farming aspects to Farmer's Delight

### DISCUSSION THREAD

**Discussion by DaCooladeMan** [vectorwing/FarmersDelight#864] [Ideas]
This post is more of a pull request than an issue, but I couldn't figure out how to post a pull request, so I'm posting it here. Farmer's Delight is a really good food mod, if not the best food mod ever created. But it doesn't really have enough farming aspects to it. More cooking. I want it to 75% farming, 25% cooking, because it's called "Farmer's Delight" not "Chef's Delight". I want there to be a watering can, which you can use to water crops, which in turn, causes crops to grow 10% or 15% faster. There also needs to be Animal Waste and stuff which you can create fertilizer with to boost crop growing speed by 25%. There can also be a rake, which allows you to create farmland in a 3 by 3 square, more efficient than a hoe. There also needs to be other fruits, like bananas. I am unsure if a Farmer's Delight addon adds bananas to the game, but if not, there definitely needs to be a banana. You can create Banana Milkshakes out of it, which gives you a 1:00 speed boost when drank. Now, obviously bananas grow on trees, which there needs to be a banana tree sapling to farm bananas. There you have it. All of my suggestions for Farmer's Delight. I think they're good suggestions, considering that there are not a lot of farming tools in Farmer's Delight. I extremely enjoy this mod, and hope that it gets improved as the days go by.

**Comment by incaplinca** [comment]
you should put suggestions in Discussions, just so actual issues don’t get covered with ideas

**Comment by MythrilBagels** [comment]
I don't think you understand what a Pull Request is...
A Pull Request is when someone wants their code imported into the main repository. Imagine there is a bug in the mod, and someone else makes a Fork of the project (copy it for themselves) to fix the bug. They then submit a pull request to get their bug fix incorporated into the main codebase.

**Comment by DaCooladeMan** [comment]
> I don't think you understand what a Pull Request is... A Pull Request is when someone wants their code imported into the main repository. Imagine there is a bug in the mod, and someone else makes a Fork of the project (copy it for themselves) to fix the bug. They then submit a pull request to get their bug fix incorporated into the main codebase.

Oh, I thought a pull request was a request to add a feature into the game. Thanks for the info!

> you should put suggestions in Discussions, just so actual issues don’t get covered with ideas

Thanks!

**Comment by DaCooladeMan** [comment]
> I don't think you understand what a Pull Request is... A Pull Request is when someone wants their code imported into the main repository. Imagine there is a bug in the mod, and someone else makes a Fork of the project (copy it for themselves) to fix the bug. They then submit a pull request to get their bug fix incorporated into the main codebase.

Oh, I thought a pull request was a request to add a feature into the game. Thanks for the info!

**Comment by underscoreffyerauhfasf** [comment]
not to speak in vector's place, but i'm speaking in vector's place: this seems like something that would be more suited towards an addon than changing the mod itself. farmer's delight doesn't require permission in order to create addons for it, so if you want to try your hand at integrating more farming into FD, you're free execute it in any way you see fit 👍

**Comment by DaCooladeMan** [comment]
> not to speak in vector's place, but i'm speaking in vector's place: this seems like something that would be more suited towards an addon than changing the mod itself. farmer's delight doesn't require permission in order to create addons for it, so if you want to try your hand at integrating more farming into FD, you're free execute it in any way you see fit 👍

Thanks. I just thought features like these would fit more into FD than into an addon for the mod. For example, Banana milkshakes. That should be in FD but it's an addon. It would just seem weird and out of place when I look at it. But I could be wrong, who knows.

**Comment by a-a-GiTHuB-a-a** [comment]
The idea of addons _is_ to add functionality—the mod does what the mod does, but at least it's extensible. If it feels like something "should" be there, that's the idea to start an addon, right?
